int32 current_person_index
---
string identified_item_name